cppcache/src/ExpiryTaskManager.cpp

```diff
@@ -51,7 +51,7 @@ void ExpiryTaskManager::start() {
   auto start_future = start_promise.get_future();
   runner_ = std::thread{[this, &start_promise] {
     start_promise.set_value(true);
-    DistributedSystemImpl::setThreadName(NC_ETM_Thread);
+    Log::setThreadName(NC_ETM_Thread);
```

Comment:
Can we just inline the string constants, they aren't used anywhere else.

Reply:
Done